app/helpers/test_helpers.go

```diff
@@ -48,21 +56,22 @@ func (EmptyAppOptions) Get(o string) interface{} { return nil }
 func Setup(t *testing.T, isCheckTx bool, invCheckPeriod uint) *gaiaapp.GaiaApp {
 	t.Helper()
 
-	app, genesisState := setup(!isCheckTx, invCheckPeriod)
-	if !isCheckTx {
-		// InitChain must be called to stop deliverState from being nil
-		stateBytes, err := json.MarshalIndent(genesisState, "", " ")
-		require.NoError(t, err)
+	privVal := mock.NewPV()
+	//todo check context here
+	pubKey, err := privVal.GetPubKey(nil)
```

Comment:
@yaruwangway I agree with the linter, could we make the recommended changes?

Also do you see any issues with the recommended approach?

Reply:
the lint issue is solved.